numpyro/infer/hmc_util.py

```diff
@@ -390,25 +393,26 @@ def init_fn(z, rng_key, step_size=1.0, inverse_mass_matrix=None, mass_matrix_siz
         return HMCAdaptState(step_size, inverse_mass_matrix, mass_matrix_sqrt,
                              ss_state, mm_state, window_idx, rng_key)
 
-    def _update_at_window_end(z, rng_key_ss, state):
+    def _update_at_window_end(z_info, rng_key_ss, state):
         step_size, inverse_mass_matrix, mass_matrix_sqrt, ss_state, mm_state, window_idx, rng_key = state
 
         if adapt_mass_matrix:
             inverse_mass_matrix, mass_matrix_sqrt = mm_final(mm_state, regularize=True)
             mm_state = mm_init(inverse_mass_matrix.shape[-1])
 
         if adapt_step_size:
-            step_size = find_reasonable_step_size(step_size, inverse_mass_matrix, z, rng_key_ss)
+            step_size = find_reasonable_step_size(step_size, inverse_mass_matrix, z_info, rng_key_ss)
             ss_state = ss_init(jnp.log(10 * step_size))
 
         return HMCAdaptState(step_size, inverse_mass_matrix, mass_matrix_sqrt,
                              ss_state, mm_state, window_idx, rng_key)
 
-    def update_fn(t, accept_prob, z, state):
+    def update_fn(t, accept_prob, z_info, state):
         """
         :param int t: The current time step.
         :param float accept_prob: Acceptance probability of the current trajectory.
-        :param z: New position drawn at the end of the current trajectory.
+        :param z_info: A tuple contains new position drawn at the end of the current trajectory,
```

Comment:
nit: can we reuse this z_info description in all the remaining docstrings too?

Reply:
I have change `z_info` tuple to IntegratorState, so we can simply say "the new integrator state" here